runtime/parachains/src/hrmp.rs

```diff
@@ -470,24 +531,35 @@ pub mod pallet {
 		///
 		/// If there are pending HRMP close channel requests, you can use this
 		/// function process all of those requests immediately.
-		#[pallet::weight(0)]
-		pub fn force_process_hrmp_close(origin: OriginFor<T>) -> DispatchResult {
+		///
+		/// Total number of closing channels must be provided as witness data.
```

Comment:
Nit: Total number of closing channels must be provided as witness data for correctly estimating the weight?

(Here and in other places)